src/main/java/org/logstash/beats/Server.java

```diff
@@ -54,12 +59,12 @@ public Server listen() throws InterruptedException {
                 logger.error("Could not shut down worker group before starting", e);
             }
         }
-        bossGroup = new NioEventLoopGroup(eventLoopThreadCount); // TODO: add a config to make it adjustable, no need many threads
-        workGroup = new NioEventLoopGroup(eventLoopThreadCount);
+        bossGroup = new NioEventLoopGroup(eventLoopThreadCount, daemonThreadFactory(id + "-bossGroup")); // TODO: add a config to make it adjustable, no need many threads
+        workGroup = new NioEventLoopGroup(eventLoopThreadCount, daemonThreadFactory(id + "-workGroup"));
```

Comment:
+1, this is helpful for investigation.

- And we should also do same to http-input: https://github.com/logstash-plugins/logstash-input-http/blob/main/src/main/java/org/logstash/plugins/inputs/http/NettyHttpServer.java#L46C77-L46C102
- set boss group thread 1 by default